scrapers/DigitalDesire.yml

```diff
@@ -0,0 +1,25 @@
+name: DigitalDesire.com
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - digitaldesire.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //meta[@property="og:title"]/@content
+     Date:
+        selector: //div[@class="set_date"]
+        postProcess:
+          - parseDate:  January 2, 2006
+     Performers:
+      Name: //div[@class="title_bar gallery_model"]/span/a
+     Tags:
+      Name: //span[@class="update_tags"]/a
+     Image:
+        selector: //meta[@property="og:image"]/@content
```

Comment:
the images returned are some horrible resizes eg `https://digitaldesire.com/tour/content/contentthumbs/48/93/44893-2x.jpg` (press the zoom if needed to see the actual image)
i would regex replace the `-2x` to `-1x` to get the original image
`https://digitaldesire.com/tour/content/contentthumbs/48/93/44893-1x.jpg`

Reply:
Yup that brings the right size in 👍